docs/integrations/webhooks/overview.mdx

```diff
@@ -1,70 +1,35 @@
 ---
 title: Webhooks overview
-description: Clerk webhooks allow you to receive event notifications from Clerk. Clerk will send a POST request to a URL you specify when certain events happen in your Clerk account.
+description: Clerk webhooks allow you to receive event notifications from Clerk.
 ---
 
 # Webhooks overview
 
-Clerk webhooks allow you to receive event notifications from Clerk. Clerk will send a `POST` request to a URL you specify when certain events happen in your Clerk account.
+A webhook is a way for an app to provide other applications with real-time information. It delivers data to other applications as it happens, meaning you get data immediately. Unlike typical APIs where you would need to poll for data very frequently to get it real-time, webhooks are event-driven and only send data when there is an event to trigger the webhook.
```

Comment:
Its important to note that a webhook is not 'real-time' or 'as its happens'. It can be quite fast, but there is a definite delay and webhooks are not useful in a real-time onboarding flow.

Webhooks are event driven, but still async and your application can't rely on real-time delivery.

Reply:
yes you are correct if you want to be extremely precise, but in general, webhooks are responsible for real-time data delivery. their entire gimmick is about being "real-time". many chat/text services use webhooks, and every one knows that when a message is sent, of course there is a slight delay for the other to receive it - nothing is ever really instant.

what text updates do you suggest if you would like to provide this information?

Reply:
what text do you suggest we write instead !